content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Sidecar Containers
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+Sidecar containers are the secondary containers that runs along with the main
+application container within the same {{< glossary_tooltip text="Pod" term_id="pod" >}}.
+These containers are used to enhance or extend the functionality of the primary container
+by providing additional services, or functionality such as logging, monitoring, security,
+or data synchronization, without directly altering the primary application code.
+
+<!-- body -->
+
+## API for sidecar containers
+
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
```

Comment:
This should move right to the top of the page now it's a standalone concept.

Reply:
I was little confused on it and updated as per my understanding, PTAL and let me know if my understanding is wrong.